app/assets/javascripts/covid19/src/data/Covid19Data.types.ts

```diff
@@ -96,14 +99,18 @@ export interface Sample {
     cellComponent?: string[];
     domains?: string[];
 }
+
 export interface Details {
     refEMDB?: RefEMDB;
     refPDB?: RefPDB;
     sample: Sample;
     refdoc: RefDoc[];
 }
-export interface RefModel {
-    method?: string;
-    externalLink: Url;
-    queryLink: Url;
+
+export interface PdbValidation {
+    method: "PDB-Redo" | "Isolde" | "Refmac";
+    externalLink?: Url;
+    queryLink?: Url;
 }
+
+export interface RefModels extends Array<PdbValidation> {}
```

Comment:
Not that this is wrong, but what's the difference with the standard `type RefModels = PdbValidation[]`?